compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/DecomposeAttention.cpp

```diff
@@ -366,6 +366,16 @@ void DecomposeAttentionPass::runOnOperation() {
     SmallVector<Operation *> ops;
     decomposeTiledAttention(attnOp, ops, rewriter, optionalTileSize);
```

Comment:
Can we remove the "decomposeTiledAttention" part now? They both are doing the same thing?

Reply:
I'd like to do that in a separate patch. There are a number of transform scripts for attention (The CUDA attention transform scripts) that I need to take into account before doing this.